### DISCUSSION TITLE
About suicide!

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1395] [General]
Hi
Can't understand, what did I do wrong?
Using Edge 1030, MTB 24.7.0 with Java runtime and import assistant:
once file is shown as imported in "Tour import" (can see a little green icon) it doesnt show up in "Tour book"! 
Thanks

**Comment by FJBDev** [comment]
> Hi Can't understand, what did I do wrong? Using Edge 1030, MTB 24.7.0 with Java runtime and import assistant: once file is shown as imported in "Tour import" (can see a little green icon) it doesnt show up in "Tour book"! Thanks

Did you make sure to save your tour in the database ,i.e.: See step 6 of the "First steps" documentation [here](https://mytourbook.sourceforge.io/mytourbook/index.php/first-steps)

**Reply by Thompson-ongithub** [reply]
could it be that you just don't show the tour type you assigned the newly imported tour to? 
In this case either choose "all tour types" or  even "type not defined" and your newly imported tour should show up.

**Comment by wolfgang-ch** [comment]
All tours in different views are filtered with the tour filters https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/workspace#app-filter
